test/oqs_test_groups.c

```diff
@@ -86,9 +86,20 @@ static int test_oqs_groups(const char *group_name, int dtls_flag) {
 
 static int test_group(const OSSL_PARAM params[], void *data) {
     int ret = 1;
-    int *errcnt = (int *)data;
+    int *errcnt = (int *)data, *mintls = NULL;
     const OSSL_PARAM *p =
-        OSSL_PARAM_locate_const(params, OSSL_CAPABILITY_TLS_GROUP_NAME);
+        OSSL_PARAM_locate_const(params, OSSL_CAPABILITY_TLS_GROUP_MIN_TLS);
+
+    if (p == NULL || p->data_type != OSSL_PARAM_INTEGER) {
+        ret = -1;
+        goto err;
+    }
+
+    mintls = (int *)p->data;
+    if (*mintls == -1)
+        return 1;
```

Comment:
This basically means "test succeeded -- but not done": Would it be even better to react on this retval in the calling function, e.g., outputting a "skipped" message (or none at all)? That would also allow a count of algs OK, NOK, Skipped.